emails/tests/mgmt_send_welcome_emails_tests.py

```diff
@@ -74,6 +76,49 @@ def test_send_welcome_emails(
     assert expected_cta in body_html
 
 
+def client_error_on_invalid_email(*args, **kwargs):
+    for address in kwargs["Destination"]["ToAddresses"]:
+        if "±×÷√" in address:
+            raise ClientError(
+                operation_name="SendEmail",
+                error_response={
+                    "Error": {
+                        "Code": "InvalidParameterValue",
+                        "Message": "Invalid email address",
+                    }
+                },
+            )
+
+
+@pytest.mark.django_db
+def test_invalid_email_address_skips_invalid(
+    mock_ses_client: MagicMock, caplog: pytest.LogCaptureFixture
+):
```

Comment:
Nit: needs return type annotation
```suggestion
) -> None:
```